### PR TITLE
Fix order total minimum value based on currencies

### PR DESCRIPTION
**Pull request by corsonr**
By default the addon stops the checkout process when the order total is below 0.50, but according to this [Stripe official document](https://support.stripe.com/questions/what-is-the-minimum-amount-i-can-charge-with-stripe), the minimum value for an order to be accepted by Stripe depends on the currency.

The fix sets a different minimum order value based on the site currency, it also formats the error message to now include the currency symbol:

@roykho 

479052-zd-woothemes

### CODE REVIEW

**Comment by glagonikas** [comment]
Hi @corsonr is it possible to apply the requested changes so we can get this bug resolved? Thanks

**Comment by corsonr** [comment]
yep, it will be done this week

**Comment by glagonikas** [comment]
Hi @corsonr is it possible to resolve this, as we need to start selling some cheaper products soon.

**Comment by corsonr** [comment]
Sorry for the delay on this one. Fixed @roykho !

File: woocommerce-gateway-stripe.php [hunk 1]
```diff
@@ -418,6 +418,49 @@ public function woocommerce_payment_token_set_default( $token_id ) {
 			$stripe_customer->set_default_card( $token->get_token() );
 		}
 	}
+	
+	/**
+	 * Checks Stripe minimum order value authorized per currency
+	 */
+	public function get_minimum_amount() {
```
Comment (roykho) [inline woocommerce-gateway-stripe.php]: You're going to get warnings because your did not explicitly declare this as static.

Comment (corsonr) [inline woocommerce-gateway-stripe.php]: weird, I didn't get any warning during my tests. Anyway, I updated the code.

**Comment by roykho** [comment]
LGTM! Thanks for the PR.

**Pull request merged**